test/e2e/service_bootstrap.py

```diff
@@ -29,9 +29,7 @@
 )
 
 
-def create_vpc() -> str:
-    region = identity.get_region()
-    ec2 = boto3.client("ec2", region_name=region)
```

Comment:
Why remove the `ec2` client? You're using the same logic to create it in your bootstrapping method.